### PR TITLE
Remove Getting A Job

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

**Review by davidthomas22 — COMMENTED** [review]
LGTM

**Review by davidthomas22 — DISMISSED** [review]
LGTM

File: server/config.js [hunk 1]
```diff
@@ -34,7 +31,6 @@ const establishments = {
     prefix: 'HMP',
     uuId: 'fd1e1db7-d0be-424a-a3a6-3b0f49e33293',
     standFirst: 'What you need to do to get or change your job in Berwyn.',
```
Comment (LucasCairns) [inline server/config.js:33]: is `.standFirst` related to GAJ? could we remove?

Comment (psoleckimoj) [reply server/config.js:33]: Good spot!!! Will remove :)

Comment (psoleckimoj) [reply server/config.js:33]: Actually, it appears to be used in the flat_content template and for that it doesn't use a path from config so I'll just remove the getting a job bits :)

Comment (LucasCairns) [reply server/config.js:33]: Awesome 💯

Comment (psoleckimoj) [reply server/config.js:33]: It's also in the basic page template.

Comment (psoleckimoj) [reply server/config.js]: Done

**Pull request merged**